elastic/apm-server
# Send system and runtime metrics to the internal metrics data stream

## Motivation/summary


## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. Run APM Server
2. Run all opbeans applications, pointed at APM Server (e.g. using apm-integration-testing with `--all-opbeans`)
3. Wait at least 1 minute to ensure all agents have sent some metrics
4. Ensure there are no service-specific application metrics data streams (`metrics-apm.app.*-*`)

## Related issues

Closes https://github.com/elastic/apm-server/issues/7520
